feat(ui): update onboarding CTA to product tour

proddata opened the pull request:
### ✨ Description

<img width="800" height="980" alt="image" src="https://github.com/user-attachments/assets/8c3c20c3-06c7-41d1-8107-5a4657472296" />

Update onboarding CTA styling and iconography in Welcome view.
- "Create my first flow” → “Start Product Tour”
- “Skip Tutorial” → “Skip Product Tour”
- Made "Create Flow" button secondary, do not distract from Product Tour CTA

### 🔗 Related Issue

none

### 🎨 Frontend Checklist

_If this PR does not include any frontend changes, delete this entire section._

- [x] Code builds without errors (`npm run build`)
- [x] All existing E2E tests pass (`npm run test:e2e`)
- [x] Screenshots or video recordings attached showing the `UI` changes

Commit:
feat(ui): update onboarding CTA to product tour

MilosPaunovic committed:
Merge branch 'develop' into proddata/onboarding-tour-cta

MilosPaunovic reviewed (approved):
From the perspective of the code, this is good to be merged.

The only thing I'd advise before merging is to run [this action](https://github.com/kestra-io/kestra/actions/workflows/auto-translate-ui-keys.yml) on `proddata/onboarding-tour-cta` branch as a base with parameter `Whether to re-translate modified keys even if they already have translations.` set to `true`.

It will open a PR to handle other languages, which you should first merge into this branch and then this one is safe to merge to `develop`.

If help needed, feel free to ping me either here or on Slack.

github-actions[bot] committed:
chore(core): localize to languages other than english (#13811)

Extended localization support by adding translations for multiple languages using English as the base. This enhances accessibility and usability for non-English-speaking users while keeping English as the source reference.

Co-authored-by: GitHub Action <actions@github.com>

akeller reviewed (approved):
Love these changes! Much clearer and keeps my focus in the middle of the screen.
